docs/standard/microservices-architecture/container-docker-introduction/docker-defined.md

```diff
@@ -7,49 +7,82 @@ ms.date: 05/26/2017
 ---
 # What is Docker?
 
-[Docker](https://www.docker.com/) is an [open-source project](https://github.com/docker/docker) for automating the deployment of applications as portable, self-sufficient containers that can run on the cloud or on-premises. Docker is also a [company](https://www.docker.com/) that promotes and evolves this technology. Docker works in collaboration with cloud, Linux, and Windows vendors, including Microsoft.
+[Docker](https://www.docker.com/) is an [open-source project](https://github.com/docker/docker) for automating the deployment of applications as portable, self-sufficient containers that can run on the cloud or on-premises. Docker is also a [company](https://www.docker.com/) that promotes and evolves this technology, working in collaboration with cloud, Linux, and Windows vendors, including Microsoft.
 
 ![](./media/image2.png)
 
 **Figure 2-2**. Docker deploys containers at all layers of the hybrid cloud
 
-Docker image containers run natively on Linux and Windows. Windows images run only on Windows hosts and Linux images run only on Linux hosts. The host is a server or a VM.
+Docker image containers can run natively on Linux and Windows. However, Windows images can run only on Windows hosts and Linux images can run only on Linux hosts, meaning host a server or a VM.
```

Comment:
I think the second sentence here is now false. The multi-architecture tags support either.

Reply:
IT's still true, since Docker can only share the base OS services. Linux would have to offer Windows services to be able to work like that.

Multi-arch is basically just a way to organize names and tags in manifest, so different images (per base OS) can use them and make it easier for us to use them, but (under the covers) there are still different images.

Reply:
I got mixed-up, Windows hosts can run Linux or Windows, only it's not because of multi-arch images but because the Hyper-V Linux VM.